src/views/editor/text.rs

```diff
@@ -721,7 +721,6 @@ pub struct SimpleStylingBuilder {
     indent_style: Option<IndentStyle>,
     tab_width: Option<usize>,
     atomic_soft_tabs: Option<bool>,
-    wrap: Option<WrapMethod>,
```

Comment:
@MinusGix I removed this since it seems its not used anymore - probably left out from the editor styles migration. Can u confirm its fine to remove it?

Reply:
Yeah looks fine to remove.